### DISCUSSION TITLE
Blocking Lava Flow outside of claims?

### DISCUSSION THREAD

**Discussion by EndaarF** [GriefPrevention/GriefPrevention#1441] [Ideas]
Is it possible to set GP so that lava only flows inside a claimed area? I have players doing a bunch of lava casting that I would really like to stop. I can turn off lava flow entirely with EssentialsXProtect, or prevent the use of lava buckets, but it would be better to be able to disable them only outside of claimed areas.

I don't see a way to do this with GP. Am I missing something, or can this be considered for a future update? Thanks.

**Comment by Jikoo** [comment]
This isn't currently a feature, but it would be possible for an addon to implement pretty easily.
